### ISSUE TITLE
How can I train the thunderNet on voc2007 datasets？

### ISSUE THREAD

**Issue by tensorflowt** [REXIMMORTAL/shufflenet-centernet#1]
Hi，I find that you have successfully trained to complete the thundernet model based on voc2007.
About the dataset, I want to know how do I prepare it ？

train_path = '/data2/intern/TF-Keras-ThunderNet/data/train.txt'
record_path = os.path.join(base_path, 'model/record.csv')

thanks！
